infra/resources/modules/commons/dashboard.tf

```diff
@@ -0,0 +1,22 @@
+resource "azurerm_dashboard" "main" {
+  name                = "Main"
+  resource_group_name = "dashboards"
```

Comment:
**note**: I kept the same namespaces we already have on the Azure portal which is located under the `identity` module